main.css

```diff
@@ -0,0 +1,83 @@
+
+/*nav*/
+h1{
+  font-family: 'Yellowtail', cursive;
+  font-size: 42px;
+}
+h3{
+  font-family: "Oswal";
+  font-size: 42px;
+  font-style: bold;
+}
+h2 h5{
+  font-size: 10px;
+  line-height: 170%;
+}
+ul
```

Comment:
Pull your brackets up to the same line as the selector. Keep your stylesheets consistently formatted.